scripts/util.js

```diff
@@ -0,0 +1,41 @@
+const tmp = require("tmp");
+const ts = require("typescript");
+const path = require("path");
+const fs = require("fs-extra");
+
+tmp.setGracefulCleanup(); // cleans tmp file on process exit
+exports.corvidDir = path.join(__dirname, "../");
+
+const prepareEmptyTypescriptProgram = configPath => {
+  const tmpDir = tmp.dirSync();
+
+  const tmpDirPath = tmpDir.name;
+
+  const tmpConfigPath = `${tmpDirPath}/tsconfig.json`;
+  const tmpConfigContent = `{"extends": "${path.resolve(
+    exports.corvidDir,
```

Comment:
You could receive an absolute path here so you don't assume that the configPath given to you is relative to a pre-known path

Reply:
Like I said this was part of the existing solution, and since my files are all tmp I choose not change this part

Reply:
https://github.com/wix-incubator/corvid-types/pull/33#discussion_r440716150